protocol/pushnotificationserver/server.go

```diff
@@ -21,6 +21,7 @@ const encryptedPayloadKeyLength = 16
 const defaultGorushURL = "https://gorush.status.im"
 
 type Config struct {
+	Enabled bool
```

Comment:
Doesn't this kind break how we do things? All config structs are in `params/config.go`, why should this one be here?

Reply:
Same for `defaultGorushURL`, should that be in `params/defaults.go`?

Reply:
I don't know, `Messenger` config is in messenger, as other configs that are more "local", generally I am not sure it's a good pattern to have everything in `params/config`. 

I think the config should be close to where it is used, for better encapsulation and clarity, I don't see why `protocol/pushnotificationclient` namespace should depend on a top level `params/config` namespace, and I'd be vary of scattering logic around.
What do you think?

Reply:
Right, so we already have exceptions. I'd say it should be one or the other, either all in `params/config.go` or close to where it's used, as you say. Since we already have a mix there's no fixing this now without doing a lot of busywork that doesn't provide much value. Ugh.

@Samyoul what do you think? I guess it's fine. I just need to give up on consistency in this package.